lib/feature/one_hot_encoder.cc

```diff
@@ -127,10 +127,10 @@ OneHotEncoderModel::load(const std::string path) {
         "/data does not exist, or it has zero or more than one file.");
   }
 
-  ifstream model_data_input(path + "/data/" + model_data_filename);
+  std::ifstream model_data_input(path + "/data/" + model_data_filename);
   std::string model_data_str((std::istreambuf_iterator<char>(model_data_input)),
                              std::istreambuf_iterator<char>());
-  llvm::Error err = model.setModelData(model_data_str);
+  llvm::Error err = model->setModelData(std::move(model_data_str));
```

Comment:
Hmm... I just realized that will be a bad idea to specify the entire model data as a string when the model data is large. This is why we replaced `toJson()` with `save()` in FLIP-173 in Flink ML.

Let's revisit this issue in a separate PR. Maybe add a TODO?